cypress/integration/Modal.spec.ts

```diff
@@ -409,22 +408,19 @@ describe('Modal', () => {
           });
         });
 
-        it('should transfer focus to the header element', () => {
-          cy.findByRole('dialog', {name: 'Delete Item'})
-            .pipe(h.modal.getTitle)
+        it('should transfer focus to the cancel button', () => {
+          cy.findByRole('button', {name: 'Cancel'})
             .should('have.focus');
         });
 
         it('should trap focus inside the modal element', () => {
```

Comment:
This section should be updated since you set the focus on the Cancel Button instead of the header when opening the Modal. 

```suggestion
               it.only('should trap focus inside the modal element', () => {
          cy.findByRole('button', {name: 'Cancel'}).should('have.focus');
          cy.tab()
            .should('contain', 'Delete')
            .tab()
            .should('contain', 'Delete Item')
            .tab()
            .should('contain', 'Cancel');
          cy.focused().should('have.text', 'Cancel');
        });
```

Reply:
Basically, the order now with this is:

Cancel > Delete > Header > Cancel > Delete > Cancel ...